.github/workflows/test.yml

```diff
@@ -6,23 +6,28 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        entry:
-          - { ruby: "2.7", grape-swagger: "1.6.1" }
-          - { ruby: "3.0", grape-swagger: "1.6.1" }
-          - { ruby: "3.1", grape-swagger: "1.6.1" }
-          - { ruby: "3.2", grape-swagger: "1.6.1" }
-          - { ruby: "3.2", grape-swagger: "2.0.2" }
-          - { ruby: "jruby-9.4.6", grape-swagger: "1.6.1" }
-          - { ruby: "jruby-9.4.6", grape-swagger: "2.0.2" }
+        ruby: ["3.1", "3.2", "jruby-9.4.6"]
+        rails: ["6.1.7", "7.2.1"]
+        grape-swagger: ["1.6.1", "2.0.2"]
+        include:
+          - ruby: "2.7"
+            rails: "6.1.7"
+            grape-swagger: "1.6.1"
+          - ruby: "3.0"
+            rails: "6.1.7"
+            grape-swagger: "1.6.1"
```

Comment:
The versions under "include" are neither compatible with Rails >=7 nor grape-swagger >=2 so we list them separately here.

Reply:
This is why we have the other style matrix here. The full exploded combination + includes is really hard to read. Want to just list things that are compatible like they were before?

Reply:
It probably boils down to a matter of preference, there's a tradeoff here: when adding support for a new version you have to remember to copy the right lines when adding a new Ruby version, whereas the approach I used here automatically tests agains all the supported versions of the other dependencies. I don't mind going with the previous approach though and will revert back to that.